src/UI/Callback.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import axios from 'axios';
+import { Redirect } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { fetchUser } from '../store/actions';
+
+const Callback = (props) => {
```

Comment:
`Callback` sounds a bit generic, isn't it? :) Name it like `OAuthCallback` or whatever